### PR TITLE
be: java intrinsics, support for `set_field`, complement of `get_field`

### LINKED ISSUE #3852: java intrinsics, support for set_field, complement of get_field

### PR DESCRIPTION
**Pull request by simonvonhackewitz**
fix #3852

### CODE REVIEW

File: src/dev/flang/be/jvm/Intrinsix.java [hunk 1]
```diff
@@ -382,6 +381,29 @@ private static Expr locked(Expr e)
           return new Pair<>(res.andThen(returnNewJavaObject(jvm, rc)), Expr.UNIT);
         });
 
+    put("fuzion.java.set_static_field0",
+        (jvm, si, cc, tvalue, args) ->
+        {
+          // var rc = jvm._fuir.clazzResultClazz(cc);
```
Comment (michaellilltokiwa) [inline src/dev/flang/be/jvm/Intrinsix.java:387]:
```suggestion
```

File: src/dev/flang/be/jvm/Intrinsix.java [hunk 2]
```diff
@@ -411,6 +433,41 @@ private static Expr locked(Expr e)
 
           return new Pair<>(res.andThen(returnNewJavaObject(jvm, rc)), Expr.UNIT);
         });
+
+    put("fuzion.java.set_field0",
+        (jvm, si, cc, tvalue, args) ->
+        {
+          // var rc = jvm._fuir.clazzResultClazz(cc);
+          // var rt = jvm._types.javaType(rc);
```
Comment (michaellilltokiwa) [inline src/dev/flang/be/jvm/Intrinsix.java:441]:
```suggestion
```

File: src/dev/flang/be/jvm/runtime/Runtime.java [hunk 3]
```diff
@@ -909,6 +909,37 @@ public static Object fuzion_java_get_static_field0(String clazz, String field)
...
+   * @param clazz name of the class of the field
+   *
+   * @param field name of the field
+   *
+   * @param value the value to which the field should be set
+   */
+  public static void fuzion_java_set_static_field0(String clazz, String field, Object value)
+  {
+    System.out.println("  debug: fuzion_java_set_static_field0");
+
```
Comment (michaellilltokiwa) [inline src/dev/flang/be/jvm/runtime/Runtime.java:926]:
```suggestion
```

File: src/dev/flang/be/jvm/runtime/Runtime.java [hunk 4]
```diff
@@ -947,6 +978,44 @@ public static Object fuzion_java_get_field0(Object thiz, String field)
...
+   *
+   * @param field name of the field
+   *
+   * @param value the value the field should be set to
+   */
+  public static void fuzion_java_set_field0(Object thiz, String field, Object value)
+  {
+
+    System.out.println("  debug: call to fuzion_java_set_field0");
+
```
Comment (michaellilltokiwa) [inline src/dev/flang/be/jvm/runtime/Runtime.java:997]:
```suggestion
```

File: src/dev/flang/be/jvm/runtime/Runtime.java (hunk 3 above)
Comment (simonvonhackewitz) [reply src/dev/flang/be/jvm/runtime/Runtime.java:926]: 🙈

**Pull request merged**